### ISSUE TITLE
Auto generated functions do not work

### ISSUE THREAD

**Issue by steffimueller** [vaadin/gwt-polymer-elements#101]
I use the latest released version 1.2.3 and tried the widget API. It compiles fine with GWT2-8-SNAPSHOT and Java8 but when I try: 

```
@UiField
PaperBadge onlineState;

... 
onlineState.setLabel("on");
onlineState.addStyleName("on");
```

None of these functions work.

**Comment by drmichaelgorski** [comment]
+1 I have the same issue. I tried it with different elements none of the functions work.

**Comment by manolo** [comment]
Does it happen only for this component and these methods ?

**Comment by steffimueller** [comment]
No it happens for all components I have tried.

**Comment by drmichaelgorski** [comment]
Me too. None of the methods of the Components is working.

**Comment by manolo** [comment]
are those methods called in the constructor? could you paste the full code of your .ui.xml and .java ?

**Comment by drmichaelgorski** [comment]
[ui.xml](https://gist.github.com/confile/b8e66125715ba0ada3d0)

[.java](https://gist.github.com/confile/814cee5a94ad5251b535)

**Comment by drmichaelgorski** [comment]
@manolo Do you have any idea what is wrong?

**Comment by ghost** [comment]
+1 That happens for me too.

**Comment by drmichaelgorski** [comment]
I created a demo project to show the problem here: 

https://github.com/confile/vaadin-polymer-test

**Comment by jllodra** [comment]
+1 same problem here.

**Comment by aadhaarm** [comment]
+1

**Comment by manolo** [comment]
I have tested this case in one project and works as expected, take a look to these lines:

https://github.com/manolo/gwt-polymer-chat-app/blob/master/src/main/java/org/gwtcon/client/Main.ui.xml#L40

https://github.com/manolo/gwt-polymer-chat-app/blob/master/src/main/java/org/gwtcon/client/Main.java#L53

https://github.com/manolo/gwt-polymer-chat-app/blob/master/src/main/java/org/gwtcon/client/Main.java#L129

**Comment by aadhaarm** [comment]
Yes I too finally used a 'PaperBadgeElement' instead of 'PaperBadge' in ui:binder file and it works correctly. Thanks.

**Comment by manolo** [comment]
with 1.7.0.0-SNAPSHOT both should work, latest version had a problem on setting properties vs attributes

**Comment by manolo** [comment]
Fixed in 1.7.0.0
